test/evaluation/subjectIsHolderEvaluationHandler.spec.ts

```diff
@@ -19,8 +19,15 @@ describe('SubjectIsHolderEvaluationHandler tests', () => {
     const evaluationClient: EvaluationClient = new EvaluationClient();
     const evaluationHandler: SubjectIsHolderEvaluationHandler = new SubjectIsHolderEvaluationHandler(evaluationClient);
     const inputCandidates: unknown = getFile('./test/dif_pe_examples/vp/vp_subject_is_holder.json');
-    const presentation: Presentation = new Presentation(inputCandidates['@context'], inputCandidates['presentation_submission'], inputCandidates['type'], inputCandidates['verifiableCredential'], inputCandidates['holder'], inputCandidates['proof']);
-    evaluationClient.verifiablePresentation = new VP(presentation);
+    const presentation: VerifiablePresentation = {
+      '@context': inputCandidates['@context'],
+      presentationSubmission: inputCandidates['presentation_submission'],
+      type: inputCandidates['type'],
+      verifiableCredential: inputCandidates['verifiableCredential'],
+      holder: inputCandidates['holder'],
+      proof: inputCandidates['proof']
+    };
+    evaluationClient.verifiablePresentation = presentation;
```

Comment:
instantiation with new is cleaner.

Reply:
Nope it is not. You are looking at this with a Java hat on